schemas/jsonschema/Quaternion.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$comment": "Generated from Quaternion by @foxglove/message-schemas",
+  "title": "Quaternion",
+  "description": "A [quaternion](https://eater.net/quaternions) representing a rotation in 3D space",
```

Comment:
I think the link is useful to have _somewhere_ in the docs? Otherwise people who don't know what quaternions are have to google it themselves, which may not turn up easily understandable resources.